lib/libc/machine/arm/armv8-m/arch_elf.c

```diff
@@ -0,0 +1,511 @@
+/****************************************************************************
```

Comment:
Does this file require the Samsung license?

Reply:
yes, it can be changed based on requirement